interfaces/builtin/content_test.go

```diff
@@ -195,11 +199,11 @@ apps:
 
 func (s *ContentSuite) TestResolveSpecialVariable(c *C) {
 	info := snaptest.MockInfo(c, "name: name", &snap.SideInfo{Revision: snap.R(42)})
-	c.Check(builtin.ResolveSpecialVariable("foo", info), Equals, "/snap/name/42/foo")
```

Comment:
How that? On Debian & Ubuntu we have /snap and on Fedora we have /var/lib/snapd/snap in and outide of the snap or do I miss anything?